doc/news/_preparation_next_release.md

```diff
@@ -146,13 +176,16 @@ Thanks to Armin Wurzinger.
 - The man pages for [`kdb change-resolver-symlink`](https://www.libelektra.org/manpages/kdb-change-resolver-symlink) and
    [`kdb change-storage-symlink`](https://www.libelektra.org/manpages/kdb-change-storage-symlink) referenced the wrong command.
    *(Lukas Winkler, René Schwaiger)*
+- The documentation for `kdb` and `kdb set` now mention the `--` option to stop option processing. This is useful for setting negative values among other things. *(Klemens Böswirth)*
```

Comment:
mentions

I would not call `--` an option/setting. (It is a special command-line argument which indicates where options/setting end.) (And we should use settings instead of options.)

Reply:
> (And we should use settings instead of options.)

I think settings is not a common word for something like `-V` or `--version`. I used the word “command line **switch**” in the latest commit. Anyway, in my opinion `option` would have worked as well too.